src/TableFunctions/TableFunctionNull.h

```diff
@@ -29,6 +30,7 @@ class TableFunctionNull : public ITableFunction
 
     String structure = "auto";
     ColumnsDescription structure_hint;
-};
 
+    ColumnsDescription default_structure = ColumnsDescription{NamesAndTypesList{{"dummy", std::make_shared<DataTypeUInt8>()}}};
```

Comment:
static const?

Reply:
static const ColumnsDescription can not be initialized here, must out of line.

Reply:
then just const

Reply:
also must be out of line.